src/components/Navigation/Pages/PageDefinitions/PageDefinitions.js

```diff
@@ -122,9 +122,9 @@ export class PageDefinitions extends UserManagedList {
               <a href="https://github.com/clearlydefined/curated-data">
                 https://github.com/clearlydefined/curated-data
               </a>
-              , ...{' '}
+              , , ...
             </li>
-            <li>a saved ClearlyDefined component list, package-lock.json, project-log.json, ... </li>
+            <li>a saved ClearlyDefined component list, package-lock.json or project-log.json</li>
```

Comment:
This string should not be changed, since it's not true that the application expects only those specific files